python-backend/requirements.txt

```diff
@@ -39,4 +39,5 @@ SQLAlchemy==1.2.10
 sqlalchemy-filters==0.8.0
 uwsgi==2.0.17.1
 uwsgitop==0.10
-Werkzeug==0.14.1
\ No newline at end of file
+Werkzeug==0.14.1
+ldap3==2.6.0
```

Comment:
Can you move this up to keep the list alphabetical?

Reply:
Done